packages/language-support/src/autocompletion/completionCoreCompletions.ts

```diff
@@ -617,22 +617,29 @@ function completeAliasName({
   }
 
   // parameters are valid values in all cases of symbolicAliasName
-  const baseSuggestions = parameterCompletions(
+  const parameterSuggestions = parameterCompletions(
```

Comment:
I think renaming this makes the code read clearer because you know at any point what `baseSuggestions` contain

Reply:
makes sense :+1: